examples/collada.js

```diff
@@ -0,0 +1,79 @@
+/* global itowns, document, GuiTools */
+// # Simple Globe viewer
+
+// Define initial camera position
+// Coordinate can be found on https://www.geoportail.gouv.fr/carte
+// setting is "coordonnée geographiques en degres decimaux"
+
+// Position near Gerbier mountain.
+var positionOnGlobe = { longitude: 4.21655, latitude: 44.84415, altitude: 2000 };
+
+// `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
+var viewerDiv = document.getElementById('viewerDiv');
+
+// Instanciate iTowns GlobeView*
+var globeView = new itowns.GlobeView(viewerDiv, positionOnGlobe);
+
+var menuGlobe = new GuiTools('menuDiv');
+
+var promiseElevation = [];
+
+menuGlobe.view = globeView;
+
+function addLayerCb(layer) {
+    return globeView.addLayer(layer).then(function addGui(la) {
+        if (la.type === 'color') {
+            menuGlobe.addImageryLayerGUI(la);
+        } else if (la.type === 'elevation') {
+            menuGlobe.addElevationLayerGUI(la);
+        }
+    });
+}
+// Add one imagery layer to the scene
+// This layer is defined in a json file but it could be defined as a plain js
+// object. See Layer* for more info.
+itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(addLayerCb);
+// Add two elevation layers.
+// These will deform iTowns globe geometry to represent terrain elevation.
+promiseElevation.push(itowns.Fetcher.json('./layers/JSONLayers/WORLD_DTM.json').then(addLayerCb));
+promiseElevation.push(itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addLayerCb));
+
+exports.view = globeView;
+exports.initialPosition = positionOnGlobe;
+
+exports.loadCollada = function loadCollada(url) {
+    var model;
+    // loading manager
+    var loadingManager = new itowns.THREE.LoadingManager(function _addModel() {
+        globeView.scene.add(model);
+        globeView.notifyChange(true);
+    });
+    // collada loader
+    var loader = new itowns.THREE.ColladaLoader(loadingManager);
+
+    // building coordinate
+    var coord = new itowns.Coordinates('EPSG:4326', 4.2165, 44.844, 1417);
```

Comment:
Placing the camera in a static fashion is good, but it's better if the house is placed directly on the floor like I explained [here](https://github.com/iTowns/itowns/pull/626#discussion_r166372874). It's a good showcase of a use case of a child layer.

Reply:
in SIG and architecture, it is better for the building to be placed at a fixed altitude.
your proposal makes more sense for a place mark

Reply:
> in SIG and architecture, it is better for the building to be placed at a fixed altitude.

Why?

In the context of a 3D viewer with an elevation, it makes more sense to place it according to the elevation. What will happen if elevation get updated by new measures? The house will either float or be underground?

Beside, `examples/` are also showcases: placing it in a fixed z like you do is easy: everybody can do it. But using child layer mechanism to do so is a good thing to show to our user, allowing them to better understand how iTowns work and use it at their advantage. I think this feature of itowns should be better advertised and that's the perfect example to it.

Reply:
Altitude is fixed and assumed in this example, otherwise the measures of the building would be false

Reply:
Why would they be false?

Reply:
A measurement of a building can't be exact if it moves

Reply:
What are you calling "measurement of a building"? Please be more specific, otherwise I need to guess what you meant :-)

Anyway, its size won't change. If you're talking about its position, then you're already placing it arbitrarily.

If you're talking about measurement between buildings, it depends on what you want. Sometimes you want them on the DEM, sometimes you will want them placed in a fixed way, that's true, but in this case, my point still stands:

> examples/ are also showcases: placing it in a fixed z like you do is easy: everybody can do it

If architecture guys uses iTowns, they would be able to place it absolutely, or following the elevation depending on their use case. It's better to showcase the latter.

Reply:
In this example altitude is fixed in a non arbitrary way and assumed.
Otherwise the altitude of the building would be wrong

Reply:
>  altitude is fixed

by who?

Reply:
I think what @gchoqueux means is that 3D objects are generally geolocated, they already have an altitude that has been measured in real life. So, changing it by projecting it on the ground isn't necessarily a good idea.

Reply:
Let's be clear, We need to handle both ways: 
- A fixed altitude passed with the model or defined once for all
- A clamp to ground option, updating the altitude of the model each time the DTM is updated.

-> This option can come in another PR!

Reply:
> 3D objects are generally geolocated

Well in this case, this model is not and placed arbitrarily.

> Let's be clear, We need to handle both ways:

yes of course, but way n°1 is just a z placement of a 3D object... No added value in that right? Just basic three.js stuff. The 2nd way is a good opportunity to showcase a powerful feature of iTowns.

> -> This option can come in another PR!

Right, I won't block any more on that, but please note that until this PR comes (if it comes), this house floats in the air during the time we wait for the elevation to be loaded :-/

Reply:
@gchoqueux Can you adjust the height to reduce the levitation effect (with DEMUtils for ex) ? Or is it just due to the DTM changing its value depending on level?

Reply:
@nosy-b you mean the altitude, or the z-scale of the building?